server/services/visitService.test.ts

```diff
@@ -33,6 +33,14 @@ describe('Visit service', () => {
     const application = TestData.applicationDto()
     const visit = TestData.visitDto()
     const orchestrationVisitDto = TestData.orchestrationVisitDto()
+    const cancelledVisit = TestData.orchestrationVisitDto({
+      outcomeStatus: 'ADMINISTRATIVE_CANCELLATION',
+      visitStatus: 'CANCELLED',
+    })
+    const pastVisit = TestData.orchestrationVisitDto({
+      startTimestamp: '2023-05-30T10:00:00',
+      endTimestamp: '2023-05-30T11:30:00',
+    })
```

Comment:
This fine, but I guess not strictly necessary - i.e. it doesn't matter if the object you're testing these with is *really* a past/cancelled visit as it's not what's covered by the assertions. It just needs to be a 'Visit-shaped' object. What you're asserting below is that the given service method calls the correct data client method.

Reply:
n.b. it would be different if the service method had any logic within it that looked at, say, `outcomeStatus`